### ISSUE TITLE
quick-fixes for a27

### ISSUE THREAD

**Issue by sl5net** [rossenburgg/godseye#12]
some quick-fixes (with reduced/less graphic elements) for use mods in a27 are pretty easy.
Maybe you could do it (soon) , eventually a branch with less features?
Example: https://github.com/sl5net/autociv/tree/a27_experimental

**Comment by rossenburgg** [comment]
Hey, I travelled for a while that’s why you don’t see me active ingame or on forums , I sure will come back soon but what do you mean by “less features? “ reduced graphics and etc? If yes that’s not the ideal way to resolve lags. I understand some pc with low hardware may lag but the mod doesn’t necessarily cause a lag. It just replaces or adds few things to the original game. Aside that most people says they mod resolves and issue where they are unable to join games or replay save issues . Can you explain to me more?

**Comment by sl5net** [comment]
> Hey, I travelled for a while that’s why you don’t see me active ingame or on forums , I sure will come back soon but what do you mean by “less features? “ reduced graphics and etc? If yes that’s not the ideal way to resolve lags. I understand some pc with low hardware may lag but the mod doesn’t necessarily cause a lag. It just replaces or adds few things to the original game. Aside that most people says they mod resolves and issue where they are unable to join games or replay save issues . Can you explain to me more?

no. has nothing to do with client pc/software. just some dont work in alpha27. 

i'm hurrying to get the techniques that are important to me up and running by about sunday, because that's when i want to start with a27 teamgames. so quality is less important to me. and that also worked very well in some cases, simply doing without the little things. that was my idea. but not bad. your mod is great, but there is no hurry so much.

**Comment by rossenburgg** [comment]
Do you recommend reducing graphic like Yekaterina’s mod? There’s much better way to get around lag, reducing graphics may only solve CPU lags for some older pcs I think , but network lags varies. Will be great if you can make all the necessary changes for A27 , I will push the latest version here where most features are added including the fix for auto login and other features that isn’t working correctly now. Also the security button triggers error anytime it’s pressed , you noticed that? I tried implementing user password change directly via xmpp ingame without having to use any clients but felt lazy :D

**Comment by sl5net** [comment]
> Do you recommend reducing graphic like Yekaterina’s mod?

no i just want get it work. thats all. same think that i did with autociv. you should look at the little changes that i did for get it run: https://github.com/sl5net/autociv/tree/a27_experimental

> Will be great if you can make all the necessary changes for A27

yes thank. if i have time. maybe i will do also such think for your lovely godsey mod. but i recognized that the /team 3v3 feature inside autociv works also pretty good.
